crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -1403,9 +1442,27 @@ type MoveValue {
 	json: JSON!
 }
 
+"""
+Module functions, by default, can only be called within the same module.
+These internal (sometimes called private) functions cannot be called from
```

Comment:
in what context would we refer to a module function by "private" vs "internal"?

Reply:
I've left a longer comment about my gripes with this passage (it's from the Move book), and a suggested change, but yes this was another issue -- we never use the term "internal", always "private".

Reply:
Indeed, I'd love us to have these stuff better explained in our docs.